patch/0022-dts-marvell-Add-DB-CN913X-Compexpress-board.patch

```diff
@@ -0,0 +1,560 @@
+From: Pavan Naregundi <pnaregundi@marvell.com>
+Date: Thu, 14 Sep 2023 04:35:53 +0000
+dts: marvell: Add DB-CN913X-Compexpress board
```

Comment:
Please generate the patch with `git format-patch`.

Reply:
patches are generated using git format-patch with commit id removed.